### PR TITLE
Update pricing.md

### PR DESCRIPTION
**Pull request by meghanelizabeth**
Add incident management description

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by marcweisman — APPROVED** [review]
👍

File: content/en/account_management/billing/pricing.md [hunk 1]
```diff
@@ -64,6 +64,13 @@ Datadog has many pricing plans to fit your needs. For more information, see the
   * Each host is allowed up to 4 profiled containers for free. Overage containers will be priced at $2 per container. Note: this allotment is aggregated across all hosts so if you have 4 containers on average across all your hosts, you will not be charged overages on host-by-host basis.
 * Datadog measures the total number of containers that are being profiled. A container is a self-contained operating environment that includes application software and limited operating system libraries and settings. Once every five minutes, Datadog records the number of unique containers you are monitoring in the Datadog Continuous Profiler service. Datadog charges monthly based on the fractional hours of monitored containers. For Continuous Profiler, Datadog only counts the containers that are running the Continuous Profiler service towards the total monitored container count.
 
+## Incident Management 
+
+* Datadog tracks the number of monthly active users who participate in incident management and response.
```
Comment (meghanelizabeth) [inline content/en/account_management/billing/pricing.md:69]: Datadog tracks the number of unique active users per month, who participate in incident management and response.

Comment (meghanelizabeth) [reply content/en/account_management/billing/pricing.md:69]: FYI @marcweisman

File: content/en/account_management/billing/pricing.md [hunk 2]
```diff
@@ -64,6 +64,13 @@ Datadog has many pricing plans to fit your needs. For more information, see the
...
 * Datadog measures the total number of containers that are being profiled. A container is a self-contained operating environment that includes application software and limited operating system libraries and settings. Once every five minutes, Datadog records the number of unique containers you are monitoring in the Datadog Continuous Profiler service. Datadog charges monthly based on the fractional hours of monitored containers. For Continuous Profiler, Datadog only counts the containers that are running the Continuous Profiler service towards the total monitored container count.
 
+## Incident Management 
+
+* Datadog tracks the number of monthly active users who participate in incident management and response. 
+ * An **active user** is only counted if they contribute comments or signals (graphs, links, etc.) to an incident. Anyone who only opens/closes an incident or simply views the incident are not counted. Additionally, these are not named seats, so you do not need to determine which specific users have access.
+
+
+
```
Comment (sarina-dd) [inline content/en/account_management/billing/pricing.md:73]:
```suggestion
```

**Review by sarina-dd — COMMENTED** [review]
Just a quick edit to remove add't line break

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**